src/mlpack/core/data/load.cpp

```diff
@@ -0,0 +1,33 @@
+#include "load.hpp"
+#include "load_impl.hpp"
+
+namespace mlpack {
+namespace data /** Functions to load and save matrices and models. */ {
+
+template bool Load<int>(const std::string&, arma::Mat<int>&, const bool, const bool);
+template bool Load<size_t>(const std::string&, arma::Mat<size_t>&, const bool, const bool);
+template bool Load<float>(const std::string&, arma::Mat<float>&, const bool, const bool);
+template bool Load<double>(const std::string&, arma::Mat<double>&, const bool, const bool);
+template bool Load<unsigned long long>(const std::string&, arma::Mat<unsigned long long>&, const bool, const bool);
```

Comment:
I wonder if this will have a problem if `size_t == unsigned long long`.  Not sure what system we would need to check to see (I think that's true on Windows?  Not certain).